install.sh

```diff
@@ -31,6 +42,10 @@ docker exec magento2-container bin/magento config:set payment/adyen_abstract/mer
 docker exec magento2-container ./n98-magerun2.phar config:store:set --encrypt payment/adyen_abstract/api_key_test "${ADYEN_API_KEY}" > /dev/null
 docker exec magento2-container bin/magento config:set payment/adyen_abstract/client_key_test "${ADYEN_CLIENT_KEY}"
 
+# Configure Express checkout module
+docker exec magento2-container bin/magento config:set payment/adyen_hpp/show_google_pay_on "1,2,3"
```

Comment:
Just wondering: What does "1,2,3" do in this context when you run this particular command inside the docker container?

Reply:
It enables the express module `button` on 3 locations:

1 => Minicart
2 => Cart
3 => Product detail